src/ts/dom.ts

```diff
@@ -562,6 +562,14 @@ export class DOM {
     }
   }
 
+  /**
+   * Removes an inline CSS property if it exists
+   * @param propertyName name of the property to remove
+   */
+  removeInlineCss(propertyName: string): void {
+    this.setCss(propertyName, null);
```

Comment:
Doesn't this leave an empty entry in the `style` attribute? Like `style="background:null"`?

There is also a function for explicitly this purpose: https://developer.mozilla.org/en-US/docs/Web/API/CSSStyleDeclaration/removeProperty